networkit/cpp/graph/test/GraphBuilderBenchmark.h

```diff
@@ -20,7 +20,7 @@ class GTestBenchmark : public testing::Test {
 public:
 	static void measureInMs(std::function<int()> func, int iterations = 1) {
 		Aux::Timer timer;
-		std::vector<uint64_t> runningTimes(iterations);
+		std::vector<long long unsigned> runningTimes(iterations);
```

Comment:
What is the problem here? Nitpick: I would prefer `unsigned long long` to `long long unsigned`.

Reply:
Thank you for asking. The underlying issue is that the old code uses printf(...) to format the output and `uint64_t` is incompatible to `%llu` -- this causes warnings. I do not recall why I fixed it this way (probably a hack, which made it into the commit; sorry!) and will change it back to `uint64_t` and replace the `printf` with `ostream`.